package.json

```diff
@@ -1,11 +1,18 @@
 {
   "name": "visual",
-    "scripts":{
+    "scripts":
+    {
     "postinstall":"npm run typings",
-    "typings":"node node_modules/typings/dist/bin.js i"
-  },
-  "devDependencies": {
-    "typings": "^1.3.2",
-    "d3": "3.5.5"
-  }
+    "pbiviz": "pbiviz",
+    "start": "pbiviz start",
+    "typings":"node node_modules/typings/dist/bin.js i",
```

Comment:
We should use 8 spaces as indent for lines at this block. Could you please format this file?